test/sharness/t0275-cid-security.sh

```diff
@@ -11,7 +11,7 @@ test_description="Cid Security"
 test_init_ipfs
 
 test_expect_success "adding using unsafe function fails with error" '
-  echo foo | test_must_fail ipfs add --hash murmur3-128 2>add_out
+  echo foo | test_must_fail ipfs add --hash shake-128 2>add_out
```

Comment:
This is a consequence of updating go-multihash which has removed the exposure of the murmur3 hash function due to some issues https://github.com/multiformats/go-multihash/issues/135